lib/plugins/plugin_utils.js

```diff
@@ -43,6 +43,16 @@ export default {
       }
     }
     done()
+  },
+  resolvePath (compiler, file, opts) {
```

Comment:
This is a pretty generic name, would help to have a more specific name or some docs describing what it is doing exactly